utils/measure/measure.py

```diff
@@ -276,7 +347,32 @@ def start(self):
             self.gzip_csv(csv_file_path)
 
     def should_resume(self, csv_file_path: str) -> bool:
-        """Check whether we are able to resume a previous measurement session"""
+        """This method checks if a CSV file already exists for the current color mode.
+        
+        If so, it asks the user if he wants to resume measurements or start over.
+
+        Parameters
```

Comment:
Parameters and returns also don't give added value here.

Reply:
That's incorrect.

Now:

![Screenshot_20220807_114300](https://user-images.githubusercontent.com/614929/183285219-eeffa724-2ac0-44ee-bb94-231f76679f2a.jpg)

If I remove the parts you mentioned:

![Screenshot_20220807_114402-1](https://user-images.githubusercontent.com/614929/183285241-21fabe49-f9d6-43c7-bfc6-8fcf7841435a.jpg)

Reply:
Not sure what you mean in this. `should_resume` returns a boolean so it is clear that we want to resume on `True` and not resume on `False`. 
So what does `True if we should resume measurements. False otherwise` gives for added value.
Same for the params. "The path of the CSV file that would be checked". The argument is called `csv_file_path`, even my mom should understand what this means ;-).

When these parameters are repeated in the docblock it makes maintenance and refactoring harder, because you also must not forget to change the docblock.
Maybe PyCharm does this automatically, but Visual Studio code won't for example (at least not with the default plugins).